dev/com.ibm.ws.microprofile.openapi_fat/fat/src/com/ibm/ws/microprofile/openapi/fat/ApplicationProcessorTest.java

```diff
@@ -38,19 +40,23 @@
 import componenttest.annotation.Server;
 import componenttest.annotation.SkipForRepeat;
 import componenttest.custom.junit.runner.FATRunner;
+import componenttest.rules.repeater.MicroProfileActions;
+import componenttest.rules.repeater.RepeatTests;
 import componenttest.topology.impl.LibertyServer;
 import componenttest.topology.utils.FATServletClient;
 import componenttest.topology.utils.HttpUtils;
 
 /**
- * Test to ensure exercise Application Processor. Here's summary of all the scenarios being tested:
- * - Deploy a single app and ensure it's documentation shows up in /openapi
- * - Deploy two apps and ensure one app's documentation shows up in /openapi
- * - Remove the app that was picked from the above scenario and ensure that the other app's documentation now shows up in /openapi
- * - Remove all apps and ensure no documentation (for any endpoint) is shown in /openapi
- * - Scenarios involving context root, host/port, servers
- * - Make a pure JAX-RS app with the ApplicationPath annotation and ensure that the annotations are scanned and a document is generated
- * - Complete flow: model, static, annotation, filter in order
+ * Test to ensure exercise Application Processor. Here's summary of all the
+ * scenarios being tested: - Deploy a single app and ensure it's documentation
+ * shows up in /openapi - Deploy two apps and ensure one app's documentation
+ * shows up in /openapi - Remove the app that was picked from the above scenario
+ * and ensure that the other app's documentation now shows up in /openapi -
+ * Remove all apps and ensure no documentation (for any endpoint) is shown in
+ * /openapi - Scenarios involving context root, host/port, servers - Make a pure
+ * JAX-RS app with the ApplicationPath annotation and ensure that the
+ * annotations are scanned and a document is generated - Complete flow: model,
+ * static, annotation, filter in order
  */
```

Comment:
Rather than switching to HTML (which is harder to read when looking at source), I switched off javadoc formatting for this project and then manually fixed up the formatting.